modules/installing-and-configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -0,0 +1,60 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-configuring-floating-action-button-as-a-dynamic-plugin_{context}"]
+= Configuring a floating action button as a dynamic plugin
+
+You can configure a dynamic plugin as a floating action button to perform actions (like opening an overlay, dialog, and so on), or open an internal or external link.
```

Comment:
We haven't tested the onClick actions like opening an overlay/dialog when the plugin is configured as a **dynamic plugin**. In the near future, we want to be able to open the Lightspeed chatbot using this floating button. But for now, can we stick to `open an internal or external link`?

cc @christoph-jerolimov